backend/dataall/modules/omics/aws/omics_client.py

```diff
@@ -0,0 +1,106 @@
+import logging
+import json
+
+from dataall.base.aws.sts import SessionHelper
+from dataall.core.environment.db.environment_repositories import EnvironmentRepository
+from dataall.modules.omics.db.omics_models import OmicsRun
+from dataall.modules.omics.db.omics_repository import OmicsRepository
+from botocore.exceptions import ClientError
+from dataall.core.environment.services.environment_service import EnvironmentService
+
+
+logger = logging.getLogger(__name__)
+
```

Comment:
The `OmicsClient` in `aws` should not contain references to the database `db` objects. Instead of querying the AWS Account id and region we should be giving it as parameter at init time.

Reply:
Fixed in https://github.com/data-dot-all/dataall/pull/954/commits/a6214b654f42fc354137236ee1c3bb14a8005bf1